src/guides/v2.3/extension-dev-guide/searching-with-repositories.md

```diff
@@ -507,3 +507,6 @@ The `di.xml` configuration file excerpt below shows how you can create a virtual
       </arguments>
   </type>
 ```
+
+{:.bs-callout-info}
+If the developer is building an EAV Model and needs to implement the `Repository::getList` method, they should use the EAV Filter Processor instead,  otherwise the filters won't be added to the collection.
```

Comment:
A couple things: Try not to use contraction in docs. It makes translations easier "won't" -> "will not". 
Double space before "otherwise".
But instead of  ", otherwise it won't..." I would like to see something like "Filters implemented X way are not added to the collection." Or "Custom filters are not added to the collection." Just a little more clarity about what should be avoided. Thanks!